common/views/components/IIIFViewer/IIIFViewer.tsx

```diff
@@ -217,22 +208,27 @@ const IIIFViewerComponent = ({
 }: IIIFViewerProps) => {
   const [gridVisible, setGridVisible] = useState(false);
   const [enhanced, setEnhanced] = useState(false);
-  const [parentManifest, setParentManifest] = useState(null);
-  const [currentManifestLabel, setCurrentManifestLabel] = useState(null);
+  const [parentManifest, setParentManifest] = useState<
+    IIIFManifest | undefined
+  >();
+  const [currentManifestLabel, setCurrentManifestLabel] = useState<
+    string | undefined
+  >();
   const [activeIndex, setActiveIndex] = useState(0);
   const [pageHeight, setPageHeight] = useState(500);
   const [pageWidth, setPageWidth] = useState(1000);
   const [showZoomed, setShowZoomed] = useState(false);
-  const [zoomInfoUrl, setZoomInfoUrl] = useState(null);
-  const [rotatedImages, setRotatedImages] = useState([]);
+  const [zoomInfoUrl, setZoomInfoUrl] = useState<string | undefined>();
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const [rotatedImages, setRotatedImages] = useState<any[]>([]);
```

Comment:
Sorry, I couldn't work out what this or `imageJson` were supposed to be and was in deep at this point